form-builder/src/main/java/com/dsc/form_builder/ChoiceState.kt

```diff
@@ -20,7 +20,7 @@ package com.dsc.form_builder
 class ChoiceState(
     name: String,
     initial: String = "",
-    validators: List<Validators>,
+    validators: List<Validators> = listOf(),
```

Comment:
Just curious, why did you add a default list of validators?

Reply:
For uniformity mainly. The rest of the fields have a default list of validators. Unless there is a specific reason to enforce validations on this class?

Reply:
Oh...it's fine.
Good catch